db/migrate/20150128160822_add_subject_ids_and_sms_ids_to_classifications.rb

```diff
@@ -0,0 +1,6 @@
+class AddSubjectIdsAndSmsIdsToClassifications < ActiveRecord::Migration
+  def change
+    remove_reference :classifications, :set_member_subject, index: true
+    add_column :classifications, :set_member_subject_ids, :integer, array: true, default: []
```

Comment:
Remove the `classifications_count` column on the `set_member_subjects` if we're not using the counters.